### PR TITLE
Unset reverse record

### PR DESCRIPTION
**Pull request by LeonmanRolls**
## List of features added/changed

-
-
-

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My code implements all the required features.

### CODE REVIEW

File: public/locales/en.json [hunk 1]
```diff
@@ -328,7 +328,8 @@
         "notSet": "Reverse record: not set",
         "selectPlaceholder": "Select your ENS name",
         "setTo": "Reverse record: Set to ",
-        "setToDifferent": "Reverse record: Set to a different name:"
+        "setToDifferent": "Reverse record: Set to a different name:",
+        "reverseRecordRemoval": "Are you sure you want to delete the reverse record?"
```
Comment (makoto) [inline public/locales/en.json:332]: Please generate new language keys by running `yarn i18n`. That should insert the key to each language

**Comment by makoto** [comment]
When the name is unset, it should show the user's address on the top left profile page but it's showing 0x00000

Screenshot 2021-06-18 at 11 26 01

**Comment by makoto** [comment]
When the name is unset it should show `example.eth` in the explanation page instead of `0x000`

Screenshot 2021-06-18 at 11 27 28

**Comment by makoto** [comment]
Please fix the failing test on CI (which is related to one of my earlier comments ) https://travis-ci.com/github/ensdomains/ens-app/builds/229695805#L1467

File: package.json [hunk 2]
```diff
@@ -60,7 +60,7 @@
     "slugify": "^1.4.6",
     "stackdriver-errors-js": "^0.8.0",
     "web3": "^1.3.0",
-    "web3modal": "^1.9.1"
+    "web3modal-dynamic-import": "1.10.1"
```
Comment (makoto) [inline package.json:63]: Can you transfer `web3modal-dynamic-import` under @ensdomains org?

File: src/index.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React, { Suspense } from 'react'
 import ReactDOM from 'react-dom'
-import 'core-js/es/object'
+// import 'core-js/es/object'
```
Comment (makoto) [inline src/index.js:3]: Please remove this if you no longer need it rather than commenting out

Comment (LeonmanRolls) [reply src/index.js:3]: this has been fixed in another PR

File: package.json (hunk 2 above)
Comment (LeonmanRolls) [reply package.json:63]: sure

**Review by makoto — APPROVED** [review]

**Pull request merged**